### ISSUE TITLE
SDK38: setOpacityTo on TouchableOpacity is undefined 

### ISSUE THREAD

**Issue by zanona** [expo/expo#9026]
### Summary of Issue (just a few sentences)
After upgrading to SDK38 `setOpacityTo` on `TouchableOpacity` seems to be `undefined`.

### Reproducible Demo

```tsx
const Component = () => {
	const ref = useRef(null);
	return <TouchableOpacity
		    ref={ref}
		    style={{position:'absolute',width:100,height:100,backgroundColor:'red'}}
		    onPress={() => ref.current?.setOpacityTo(0.4)}/>;

};
```

### Steps to Reproduce
- Tap the red square
- Error displayed: `_ref$current.setOpacityTo is not a function`

### Expected Behavior vs Actual Behavior
I would expect `setOpacityTo` to work as in SDK37

**Comment by AdamJNavarro** [comment]
Hey @zanona,

Given that `TouchableOpacity` is a core component from React Native, the best course of action would be to open an issue with a reproducible example in the React Native [repo](https://github.com/facebook/react-native/issues). If you're unable to reproduce it with a vanilla react native project, we can re-open this issue and explore why it's only occurring within an Expo project.

Cheers,
Adam

**Comment by zanona** [comment]
Thanks, @AdamJNavarro. Sorry, it was a false alarm. Probably something didn't cache very well across the upgrade process.
It works fine here: https://snack.expo.io/@zanona/3cb338

**Comment by zanona** [comment]
Hi again @AdamJNavarro,
I think I got confused because the snack above was working on the web version but I have now tried loading on my mobile and it indeed crashes with the same error as mentioned previously.

I don't have a way to test this on a vanilla react native project, I'm afraid. That's why I'm using Expo. :)
But I can confirm before upgrading to SDK 38 the issues didn't occur.  

So I would like to vote for reopening this issue, if possible.

**Comment by d4rky-pl** [comment]
@zanona this is an issue with React Native, they seemed to have refactored a `TouchableOpacity` component and changed `setOpacityTo` to be an internal function. Considering the docs still refer to the previous method, I assume it was not an intentional change but I may be wrong and they might've decided otherwise. In any case, I took the liberty of reusing your Snack and reported the issue where it belongs since I just ran into exactly the same problem.
